.github/workflows/conformance.yml

```diff
@@ -0,0 +1,80 @@
+name: Manually run upstream conformance test on Linux
+
+on:
+  workflow_dispatch:
+    inputs:
+      antrea-version:
+        description: The Antrea version to test. It could be a SHA-1 value, a branch, or a tag (e.g. a7b012b, release-1.12, v1.12.0).
+        default: main
+        required: true
```

Comment:
Does it make sense to provide a default and set `required: true`?

Reply:
I set required to avoid empty value being used, though the workflow will just use main in that case.
Anyway I have changed to allow empty and added comment for what code will be tested if empty, like the k8s-version option.

To reflect the versions in the workflow name, I replaced workflow name with run-name so we can know which test, which Antrea version, and which K8s version are tested in a run from the list page.

Example: https://github.com/tnqn/antrea/actions/workflows/conformance.yml